packages/client/src/__tests__/auth.test.ts

```diff
@@ -2,103 +2,59 @@
  * @vitest-environment node
  */
 
-import { describe, expect } from "vitest";
+import { afterEach, describe, expect } from "vitest";
 import {
   httpsCallable,
   HttpsCallableResult,
   FunctionsError,
 } from "@firebase/functions";
 
-import { Collection, AuthStatus, HTTPSErrors } from "@eisbuk/shared";
+import { AuthStatus, HTTPSErrors } from "@eisbuk/shared";
 import { CloudFunction } from "@eisbuk/shared/ui";
 
-import { adminDb, functions } from "@/__testSetup__/firestoreSetup";
+import { adminDb, auth, functions } from "@/__testSetup__/firestoreSetup";
 
 import { getCustomerDocPath } from "@/utils/firestore";
 
 import { testWithEmulator } from "@/__testUtils__/envUtils";
 
 import { saul } from "@eisbuk/testing/customers";
 import { setUpOrganization } from "@/__testSetup__/node";
+import { signInEmail } from "@/__testUtils__/auth";
 
 describe("Test authentication", () => {
+  afterEach(() => {
+    auth.signOut();
+  });
+
   describe("Test queryAuthStatus", () => {
-    const queryAuthStatus = (organization: string, authString: string) =>
+    const queryAuthStatus = (organization: string) =>
```

Comment:
We should probably [also change the invocation](https://github.com/eisbuk/EisBuk/blob/711f4120955f0677096a8b1957f3fee3cf9f3510/packages/client/src/store/actions/authOperations.ts#L67) of this cloud function to omit `authString` for consistency.